### DISCUSSION TITLE
[Unstable] Sprite "Content/Items/Diving/slipsuit.png" not found!

### DISCUSSION THREAD

**Discussion by ubertpendragon** [FakeFishGames/Barotrauma#15169] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

![image](https://github.com/user-attachments/assets/df33192b-f089-4131-8294-3228c432c6e3)

### Reproduction steps

Slipsuit

### Bug prevalence

Just once

### Single player or multiplayer?

Single player

### Version

v1.7.0.1 (Unstable)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Fixed in https://github.com/FakeFishGames/Barotrauma-development/commit/6e67a29bb138e658941b0b1bb8f7801802278cc2, fix coming up in the next unstable build.
